src/services/types.ts

```diff
@@ -242,7 +250,7 @@ namespace ts {
         getEncodedSyntacticClassifications(fileName: string, span: TextSpan): Classifications;
         getEncodedSemanticClassifications(fileName: string, span: TextSpan): Classifications;
 
-        getCompletionsAtPosition(fileName: string, position: number, options: GetCompletionsAtPositionOptions | undefined): CompletionInfo;
+        getCompletionsAtPosition(fileName: string, position: number, options: GetCompletionsAtPositionOptions | undefined, settings: ServicesSettings | undefined): CompletionInfo;

```

Comment:
Are callers likely to want to pass different values for `GetCompletionsAtPositionOptions` each time they call `getCompletionsAtPosition`?  Is there some way we can incorporate those options into `ServicesSettings`?